pkg/abi/linux/fuse.go

```diff
@@ -247,6 +263,63 @@ type FUSEInitOut struct {
 	_ [8]uint32
 }
 
+// FUSEInitRes is a variable-length wrapper of FUSEInitOut. The FUSE server may
+// implement older version of FUSE protocol, which contains a FUSEInitOut with
+// less attributes.
+//
+// Dynamically-sized objects cannot be marshalled.
+type FUSEInitRes struct {
```

Comment:
The purpose of `FUSEInitRes` struct is different from other variable length payload. It is used to address the version issue described in #3707, so I intended to name it in different style. It was moved to `fuse` package because it is a struct only used in processing init reply, not an actual Linux abi.